### PR TITLE
STAC-15121 - Fix pipeline

### PR DESCRIPTION
**Pull request by julianosk**
### Step 3: Did you add / update tests for your changes in the right area?
- [ ] Unit/Component test
- [ ] Integration test
- [ ] E2E/Molecule test

### Step 4: I'm confident that everything is properly tested:
I got a PO / QA Approval by:
- [ ] Name

### Step 5: Did you add release notes describing the changes you made?
- [ ] Yes

### Step 6: Can we ship this feature to production?
- [ ] Yes, I'm proud of my work. Ship it! :ship:

### CODE REVIEW

File: local_build.sh [hunk 1]
```diff
@@ -26,9 +26,9 @@ if [[ "$#" -eq "1" ]]; then
         MOUNT="/stackstate-agent-mount"
 
         docker run --rm \
-            -e ARTIFACTORY_USER=$ARTIFACTORY_USER \
-            -e ARTIFACTORY_PASSWORD=$ARTIFACTORY_PASSWORD \
-            -e ARTIFACTORY_URL="artifactory.stackstate.io/artifactory/api/pypi/pypi-local" \
+            -e artifactory_user=$artifactory_user \
+            -e artifactory_password=$artifactory_password \
+            -e ARTIFACTORY_PYPI_URL="artifactory.tooling.stackstate.io/artifactory/api/pypi/pypi-local" \
```
Comment (hruhek) [inline local_build.sh:31]: This should be `artifactory.tooling.stackstate.io/artifactory/api/pypi/pypi-virtual`.

File: .assistance/Vagrantfile [hunk 2]
```diff
@@ -45,9 +45,9 @@ Vagrant.configure("2") do |config|
     echo "export GOLANG_VERSION=1.14" >> /home/vagrant/.env
     echo "export RVM_VERSION=1.29.12" >> /home/vagrant/.env
     echo "export RUBY_VERSION=2.4.2" >> /home/vagrant/.env
-    echo "export ARTIFACTORY_USER=user" >> /home/vagrant/.env
-    echo "export ARTIFACTORY_PASSWORD=ZPxcNe6dXZmGmCz3" >> /home/vagrant/.env
-    echo "export ARTIFACTORY_URL=artifactory.stackstate.io/artifactory/api/pypi/pypi-local" >> /home/vagrant/.env
+    echo "export artifactory_user=user" >> /home/vagrant/.env
+    echo "export artifactory_password=ZPxcNe6dXZmGmCz3" >> /home/vagrant/.env
+    echo "export ARTIFACTORY_PYPI_URL=artifactory.tooling.stackstate.io/artifactory/api/pypi/pypi-local" >> /home/vagrant/.env
```
Comment (hruhek) [inline .assistance/Vagrantfile:50]: This should be `artifactory.tooling.stackstate.io/artifactory/api/pypi/pypi-virtual`

**Review by hruhek — CHANGES_REQUESTED** [review]
Two changes for PyPI url

File: .gitlab-scripts/setup_artifactory.sh [hunk 3]
```diff
@@ -1,4 +1,4 @@
 ## python artifactory dependency
 mkdir ~/.pip/ && touch ~/.pip/pip.conf
 echo "[global]" > ~/.pip/pip.conf
-echo "extra-index-url = https://$ARTIFACTORY_USER:$ARTIFACTORY_PASSWORD@$ARTIFACTORY_URL/simple" >> ~/.pip/pip.conf
+echo "extra-index-url = https://$artifactory_user:$artifactory_password@$ARTIFACTORY_PYPI_URL/simple" >> ~/.pip/pip.conf
```
Comment (hruhek) [inline .gitlab-scripts/setup_artifactory.sh:4]: `/simple` is already included in `ARTIFACTORY_PYPI_URL`

**Review by hruhek — CHANGES_REQUESTED** [review]

**Review by hruhek — APPROVED** [review]
Looks OK now :)

File: .assistance/Vagrantfile [hunk 4]
```diff
@@ -45,9 +45,9 @@ Vagrant.configure("2") do |config|
     echo "export GOLANG_VERSION=1.14" >> /home/vagrant/.env
     echo "export RVM_VERSION=1.29.12" >> /home/vagrant/.env
     echo "export RUBY_VERSION=2.4.2" >> /home/vagrant/.env
-    echo "export ARTIFACTORY_USER=user" >> /home/vagrant/.env
-    echo "export ARTIFACTORY_PASSWORD=ZPxcNe6dXZmGmCz3" >> /home/vagrant/.env
-    echo "export ARTIFACTORY_URL=artifactory.stackstate.io/artifactory/api/pypi/pypi-local" >> /home/vagrant/.env
+    echo "export artifactory_user=user" >> /home/vagrant/.env
+    echo "export artifactory_password=ZPxcNe6dXZmGmCz3" >> /home/vagrant/.env
```
Comment (Q69K) [inline .assistance/Vagrantfile:49]:
```suggestion
```

File: tasks/go.py [hunk 5]
```diff
@@ -182,7 +182,7 @@ def golangci_lint(ctx, targets, rtloader_root=None, build_tags=None):
     for target in targets:
         print("running golangci on {}".format(target))
         ctx.run(
-            "golangci-lint run -c .golangci.yml --build-tags '{}' {}".format(" ".join(tags), "{}/...".format(target)),
+            "golangci-lint run -c -v .golangci.yml --build-tags '{}' {}".format(" ".join(tags), "{}/...".format(target)),
```
Comment (Q69K) [inline tasks/go.py:185]:
```suggestion
            "golangci-lint run -v -c .golangci.yml --build-tags '{}' {}".format(" ".join(tags), "{}/...".format(target)),
```

File: .gitlab-ci-agent.yml [hunk 6]
```diff
@@ -230,14 +233,15 @@ unit_tests:
     - gofmt -l -w -s ./pkg ./cmd
     - inv -e rtloader.test
     - inv -e test --coverage --race --profile --fail-on-fmt --cpus 4 --major-version $MAJOR_VERSION --python-runtimes $PYTHON_RUNTIMES
-  tags:
-    - sts-aws
   retry:
```
Comment (Q69K) [inline .gitlab-ci-agent.yml:236]:
```suggestion
  tags:
    - sts-k8s-xl-runner
  retry:
```

File: .gitlab-ci-agent.yml [hunk 7]
```diff
@@ -250,8 +254,6 @@ unit_tests_cluster_agent:
     # TODO: --python-runtimes "$PYTHON_RUNTIMES"
     - inv -e rtloader.test
     - inv -e test --coverage --race --profile --fail-on-fmt --cpus 4 --major-version $MAJOR_VERSION --python-runtimes $PYTHON_RUNTIMES --build-include docker,linux,kubelet,kubeapiserver,python,orchestrator,clusterchecks,zlib
-  tags:
-    - sts-aws
   rules:
```
Comment (Q69K) [inline .gitlab-ci-agent.yml:257]:
```suggestion
  tags:
    - sts-k8s-xl-runner
  rules:
```

File: .gitlab-ci-cluster-agent.yml [hunk 8]
```diff
@@ -91,8 +94,6 @@ unit_tests_cluster_agent:
     # TODO: --python-runtimes "$PYTHON_RUNTIMES"
     - inv -e rtloader.test
     - inv -e test --coverage --race --profile --fail-on-fmt --cpus 4 --major-version $MAJOR_VERSION --python-runtimes $PYTHON_RUNTIMES --build-include docker,linux,kubelet,kubeapiserver,python,orchestrator,clusterchecks,zlib
-  tags:
-    - sts-aws
   retry:
```
Comment (Q69K) [inline .gitlab-ci-cluster-agent.yml:97]:
```suggestion
  tags:
    - sts-k8s-xl-runner
  retry:
```

**Pull request closed without merging**